microsoft-sharepoint-2010/0.3.9/modules/ROOT/pages/index.adoc

```diff
@@ -5,6 +5,16 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 
 Microsoft SharePoint 2010 Connector v1.0
 
+[NOTE]
+As of October 2020, Microsoft stopped providing support for Sharepoint 2010. https://techcommunity.microsoft.com/t5/microsoft-sharepoint-blog/extended-support-for-sharepoint-server-2010-ends-in-october-2020/ba-p/272628#:~:text=Mainstream%20support%20for%20SharePoint%20Server,support%20for%20SharePoint%20Server%202010[Extended support for SharePoint Server 2010 ends in October 2020]. As a result, Anypoint Connector for Sharepoint 2010 is deprecated since the underlying technology does not have support from Microsoft.
```

Comment:
It is a bit weird that the note text is in the present tense. Maybe say "Extended support for SharePoint Server 2010 ended in October 2020" for the link text.